res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -24,6 +24,9 @@ var ShiftFilterFX4 = true;
 // allow pitch bend with wheel when wheel is not active
 var PitchBendOnWheelOff = true;
 
+// use mixxx softtakeover implementation instead of the script implemented method
+// suggestion: true
+var UseEngineSoftTakeOver = true;
```

Comment:
Why you think an option is needed here? Shouldn't we just replace the old soft takeover functionality by engine.softTakeover?

Reply:
No real reason, my first changes so as a new guy on the playground i tried to not be a "hearthless jerk" and remove someone else's work entirely with my first or. :)

My honest self thought / thinks the same thing, old soft takeover makes no sense and it must go :)

Reply:
Gonna do some other fixes, gonna fix the browse button and replace / update to use 2.4 features if needed(probably tomorrow)

Should I close this PR or can I add a few more commit?